infrastructure/prod.tfvars

```diff
@@ -2,3 +2,6 @@
 redis_sku_name = "Premium"
 redis_family   = "P"
 redis_capacity = "2"
+
+# Welsh Language Reporting
+welsh_reporting_enabled = true
```

Comment:
do we need to do this? as you have default value as true for this in infrastructure/variables.tf